azure-functions-extension-fastapi/samples/fastapi_samples_streaming_download/local.settings.json

```diff
@@ -0,0 +1,8 @@
+{
+  "IsEncrypted": false,
+  "Values": {
+    "FUNCTIONS_WORKER_RUNTIME": "python",
+    "AzureWebJobsStorage": "",
+    "AzureWebJobsFeatureFlags": "EnableWorkerIndexing"
```

Comment:
Remove this. 
Also Add UseDevelopmentStorage=true for WebJobs

Reply:
will remove